prebuild/action.yml

```diff
@@ -5,12 +5,13 @@ runs:
     - name: Install node-gyp npm
       if: ${{ env.PACKAGE_MANAGER == 'npm' }}
       run: |
-        $PACKAGE_MANAGER install -g node-gyp
+        npm config set unsafe-perm true
```

Comment:
why `npm` and not `$PACKAGE_MANAGER` if it is used as `PACKAGE_MANAGER` in other lines?

Reply:
Since it's checked in the `if` statement it's known that the chosen package manager will be npm so I think is clearer to use npm or yarn from that point but, anyway, just a personal preference